File: clases/26.2-contador-servidor.py
from fastapi import FastAPI
import uvicorn

app = FastAPI()
contador = 0


@app.get("/contador")
def leer_contador():
    return {"contador": contador}


@app.put("/contador")
def incrementar_contador():
    global contador
    contador += 1
    return {"contador": contador}


@app.delete("/contador")
def borrar_contador():
    global contador
    contador = 0
    return {"contador": contador}


uvicorn.run(app, host="localhost", port=5001)

File: clases/26.2-contador-servidor.js
const express = require("express");

const app = express();
let contador = 0;

app.get("/contador", (req, res) => {
  res.json({ contador });
});

app.put("/contador", (req, res) => {
  contador++;
  res.json({ contador });
});

app.delete("/contador", (req, res) => {
  contador = 0;
  res.json({ contador });
});

app.listen(5001);
